Runtime/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationResult.java

```diff
@@ -33,6 +36,7 @@ private ConfigurationResult(Parcel in) {
   @Override
   public void writeToParcel(Parcel dest, int flags) {
     dest.writeDouble(enablePercent);
+    dest.writeString(String.valueOf(enableNonMetricMeasurement));
```

Comment:
could just write it as a `byte` 0 or 1 instead of string.
this is micro optimization, but seems silly to make it a `4 * char_encoding_size_in_bytes` instead of constant 1 byte.

Reply:
I had originally tried using a byte, but couldn't get it to pass in the unit tests.

I just looked again, and realized it's because it's using a `ParcelShadow` class. So I'll update the shadow so that this will work, but is there any reason it's using a shadow instead of just the actual `Parcel`?

Reply:
Because the android code implementing the parcelling is native code, which does not run on the host JVM - so in comes robolectric replacing it with a shadow :/

Reply:
But since the tests are run with `RobolectricTestRunner` shouldn't Parcels be shadowed automatically instead of needing a custom shadow class?

Reply:
yeah probably should, I don't rememeber why we needed that, that history got lost when we migrated to github.
if you can make it work without the custom shadow good riddance!

Reply:
sounds like a better solution than the half-assed custom parcellable we use.